### PR TITLE
Fixes #8986 . Fix backup timestamp language

### LINKED ISSUE #8986: Backup timestamp only displays in locale.us not default language
- [x] I have searched open and closed issues for duplicates
- [ ] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The backup time is displayed under "Settings -> Chats and Media -> Backups" in English in every other language setting.

### Steps to reproduce
- set a language different from english
- go to "Settings -> Chats and Media -> Backups"
- look at the displayed time of "Last backup"

**Actual result:** the "Last backup" time is displayed in english even if an other language is set in Signal
**Expected result:** the "Last backup" time is displayed in the default language of Signal

### Screenshots
Screenshot_20190814-152657
Screenshot_20190816-070524

### Device info
**Signal version:** 4.45.2

should be this source code:

Signal-Android/src/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java

private void setBackupSummary() {
    findPreference(TextSecurePreferences.BACKUP_NOW)
        .setSummary(String.format(getString(R.string.ChatsPreferenceFragment_last_backup_s), BackupUtil.getLastBackupTime(getContext(), Locale.US)));
  }

### PR DESCRIPTION
**Pull request by leethater**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Redmi Note 7, Android 9.0
 * Virtual Pixel 2, Android 9.0
 * Virtual Pixel 3, Android 9.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### Description

The backup timestamp is now written in the app default language, instead of always being in English. 
Before : (Signal is in French, but timestamp is in English)
before
After : 
after2

### CODE REVIEW

File: app/src/main/java/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public void onEvent(BackupEvent event) {
 
   private void setBackupSummary() {
     findPreference(TextSecurePreferences.BACKUP_NOW)
-        .setSummary(String.format(getString(R.string.ChatsPreferenceFragment_last_backup_s), BackupUtil.getLastBackupTime(getContext(), Locale.US)));
+        .setSummary(String.format(getString(R.string.ChatsPreferenceFragment_last_backup_s), BackupUtil.getLastBackupTime(getContext(), getResources().getConfiguration().locale)));
```
Comment (alan-signal) [inline app/src/main/java/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java:103]: Thanks, can you try `Locale.getDefault()` and see if that works too. That's what we commonly use.

Comment (leethater) [reply app/src/main/java/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java]: Thanks, it works with your proposition.

**Comment by Meteor0id** [comment]
Please consider #9403
And I think that #8986 is a duplicate of #8842

**Comment by alan-signal** [comment]
Merged in 88a40be9015e9265221bd637158a23e8d02781c1

Thank you!

**Pull request closed without merging**